shared/fs/filepreview/view-container.js

```diff
@@ -28,7 +28,7 @@ const mapStateToProps = (state: TypedState, {path}: Props) => {
   const _pathItem = state.fs.pathItems.get(path, Constants.makeFile())
   return {
     _serverInfo: state.fs.localHTTPServerInfo,
-    mimeType: _pathItem.type === 'file' ? _pathItem.mimeType : '',
+    mimeType: _pathItem.type === 'file' ? _pathItem.mimeType : null,
     isSymlink: _pathItem.type === 'symlink',
   }
```

Comment:
Hmm this must have been my fault originally. But since you're here, would you mind replacing `mimeType` and `isSymlink` with `_pathItem`, and do those in `mergeProps`? The reason is `mapStateToProps should be as fast as possible, as long as the change doesn't cause re-rendering more. In this case, being contingent on a single `pathItem` is infrequent enough I think.